lib/validation/validators.js

```diff
@@ -42,24 +42,24 @@ module.exports = Validators = {
     },
 
     numeric: function numeric(value) {
-        return Validators.regex(value, /^\d+$/);
+        return Validators.regex(value, /^\d*$/);
     },
 
     equal: function equal(value) {
         var values = [].slice.call(arguments, 1);
-        return values.length && _.any(values, function (c) { return value === c; });
+        return values.length && _.any(values, function (c) { return value === c || value === ''; });
```

Comment:
The `|| value === ''` should probably live outside the `_.any` call. We don't need to the check the value against an empty string for every possible value.

`return values.length && (value === '' || _.any(values, function (c) { return value === c; }));`